sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachinesOperations.cs

```diff
@@ -329,16 +329,19 @@ public async Task<AzureOperationResponse<VirtualMachine>> UpdateWithHttpMessages
         /// <param name='vmName'>
         /// The name of the virtual machine.
         /// </param>
+        /// <param name='forceDeletion'>
+        /// Optional parameter to force delete virtual machines.(Feature in Preview)
+        /// </param>
         /// <param name='customHeaders'>
         /// The headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string vmName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string vmName, bool? forceDeletion = default(bool?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is also a breakign change that can be mitigated with an additional partial class.  Here a is an example of how this can be mitigated: 

``` c# 
public partial class VirtualMachineOperations
{
  public Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string vmName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
  {
    return DeleteWithHttpMessagesAsync( resourceGroupName, vmName, false, customHeaders, cancellationToken);
  }
```
}

Reply:
Added customizations as suggested.